main/src/ca/uwaterloo/flix/language/phase/EntryPoints.scala

```diff
@@ -513,52 +513,36 @@ object EntryPoints {
 
   }
 
-  private object WrapInDefaultHandlers {
-    /**
-      * Default handler components
-      *
-      * Given:
-      * {{{
-      *     eff E
-      *     mod E {
-      *         @DefaultHandler
-      *         def handle(f: Unit -> a \ ef): a \ (ef - E) + IO = exp
-      *     }
-      * }}}
-      *
-      *   - handlerSym: handle's [[Symbol.DefnSym]].
-      *   - handlerDef: handle's [[TypedAst.Def]].
-      *   - handlerDef: E's [[Type]].
-      *   - handlerDef: E's [[Symbol.EffSym]].
-      * */
-    private case class DefaultHandler(
-                                       handlerSym: Symbol.DefnSym,
-                                       handlerDef: TypedAst.Def,
-                                       handledEff: Type,
-                                       handledSym: Symbol.EffSym,
-                                     )
+  /**
+    * Default handler components
+    *
+    * Given:
+    * {{{
+    *     eff E
+    *     mod E {
+    *         @DefaultHandler
+    *         def handle(f: Unit -> a \ ef): a \ (ef - E) + IO = exp
+    *     }
+    * }}}
+    *
+    *   - handlerSym: handle's [[Symbol.DefnSym]].
+    *   - handlerDef: handle's [[TypedAst.Def]].
+    *   - handlerDef: E's [[Type]].
+    *   - handlerDef: E's [[Symbol.EffSym]].
+    * */
+  case class DefaultHandler(
+                             handlerSym: Symbol.DefnSym,
+                             handlerDef: TypedAst.Def,
+                             handledEff: Type,
+                             handledSym: Symbol.EffSym,
+                           )
+
+  private object CheckDefaultHandlers {
```

Comment:
I am not really sure why we even need these private objects. I think we could just flatten the EntryPoints namespace?